ui/src/utils/guard.ts

```diff
@@ -398,6 +399,7 @@ export const initAppSettingsStore = async () => {
     customizeStore.getState().update(appSettings.custom_css_html);
     themeSettingStore.getState().update(appSettings.theme);
     seoSettingStore.getState().update(appSettings.site_seo);
+    writeSettingStore.getState().update(appSettings.site_write);
```

Comment:
site_write only returns the required information

Reply:
only update `restrict_answer`, does it meet the requirement?